client/src/main/scala/org/http4s/client/Client.scala

```diff
@@ -96,7 +117,7 @@ final case class Client(open: Service[Request, DisposableResponse], shutdown: Ta
     }
   }
 
-  @deprecated("Use fetchAs", "0.12")
+  @deprecated("Use expect", "0.14")
```

Comment:
Seems like a reasonable strategy. I imagine we will aggressively prune deprecated interfaces before a 1.0 release.